### PR TITLE
added ambient rotate for other angles - theta, phi, gamma (take 2)

### PR DESCRIPTION
**Pull request by dor2727**
Allowing 3D ambient_camera_rotation around other angles - theta, phi, and gamma

merged with the current master, and blacked

### CODE REVIEW

File: manim/scene/three_d_scene.py [hunk 1]
```diff
@@ -71,20 +71,34 @@ def begin_ambient_camera_rotation(self, rate=0.02):
...
+
+    def stop_ambient_camera_rotation(self, about="theta"):
         """
         This method stops all ambient camera rotation.
         """
-        self.renderer.camera.theta_tracker.clear_updaters()
-        self.remove(self.renderer.camera.theta_tracker)
+        if about == "phi":
+            x = self.renderer.camera.phi_tracker
+        elif about == "gamme":
```
Comment (leotrs) [inline manim/scene/three_d_scene.py:95]: Typo: gamme

**Review by leotrs — CHANGES_REQUESTED** [review]

File: manim/scene/three_d_scene.py [hunk 2]
```diff
@@ -71,20 +71,34 @@ def begin_ambient_camera_rotation(self, rate=0.02):
...
+            x = self.renderer.camera.gamma_tracker
+        else:
+            x = self.renderer.camera.theta_tracker
```
Comment (huguesdevimeux) [inline manim/scene/three_d_scene.py:98]: Same comment as above:)

File: manim/scene/three_d_scene.py [hunk 3]
```diff
@@ -71,20 +71,34 @@ def begin_ambient_camera_rotation(self, rate=0.02):
...
-        self.renderer.camera.theta_tracker.add_updater(
-            lambda m, dt: m.increment_value(rate * dt)
-        )
-        self.add(self.renderer.camera.theta_tracker)
+        if about == "phi":
+            x = self.renderer.camera.phi_tracker
+        elif about == "gamme":
+            x = self.renderer.camera.gamma_tracker
+        else:
+            x = self.renderer.camera.theta_tracker
```
Comment (huguesdevimeux) [inline manim/scene/three_d_scene.py:84]: **Issue :** Why not raising an Exeption in this case ?

**Review by huguesdevimeux — CHANGES_REQUESTED** [review]
Thanks for the contribution! 

I left a suggestion.

File: manim/scene/three_d_scene.py (hunk 3 above)
Comment (dor2727) [reply manim/scene/three_d_scene.py:84]: Well, for no good reason. You are right.

**Review by huguesdevimeux — APPROVED** [review]
SGTM.
Thanks for this !

**Comment by kolibril13** [comment]
Nice, this is great!
Can you add these new roations  for gamma and theta to this example?
https://docs.manim.community/en/latest/examples/3d.html#threedcamerarotation

**Comment by leotrs** [comment]
Hello @dor2727, any updates on this branch?

**Review by kolibril13 — APPROVED** [review]

**Review by leotrs — APPROVED** [review]

**Pull request merged**